tests/runtime/basic

```diff
@@ -158,6 +158,16 @@ EXPECT hardware:cache-misses:
 EXPECT tracepoint:xdp:mem_connect
 TIMEOUT 5
 
+NAME it lists given search pattern
+RUN {{BPFTRACE}} -l hardware:cache-misses
+EXPECT hardware:cache-misses:
+TIMEOUT 5
+
+NAME it lists given file pattern
+RUN {{BPFTRACE}} -l hardware-cache-misses.bt
+EXPECT_REGEX No such file or directory
```

Comment:
Nit: Could you include the full error message and `EXPECT` instead of `EXPECT_REGEX`?

We've had problems in the past where partial expected outputs were hiding real bugs, so are trying to move away from them where possible